lisa/tools/lscpu.py

```diff
@@ -26,6 +28,17 @@ def _windows_tool(cls) -> Optional[Type[Tool]]:
     def _check_exists(self) -> bool:
         return True
 
+    def get_architecture(self, force_run: bool = False) -> str:
+        result = self.run(force_run=force_run)
+        matched = self.__architecture.findall(result.stdout)
+        assert_that(
+            len(matched),
+            f"architecture should have exact one line, but got {matched}",
+        ).is_equal_to(1)
+        self._architecture = matched[0]
+
+        return self._architecture
```

Comment:
`self._architecture` doesn't need be an instance member here, because it's not cached, or reused. Use a local variable.